Jenkinsfile

```diff
@@ -0,0 +1,119 @@
+import groovy.json.JsonSlurper
+// This Jenkinsfile is used by Jenkins to run the 'GOUpdate' step of Reactome's release.
```

Comment:
For consistency with UniProt Update, you can separate "GO" and "Update" here, also.